### PR TITLE
Updates from Wip/multilanguage

### PR DESCRIPTION
**Pull request by Mehrad0711**
It is time for another PR!

- adding the option to batch semantically-equivalent sentences from different languages 
- optional encoder loss (forces low std on same sentences encodings)

### CODE REVIEW

**Comment by gcampax** [comment]
Not this week. This week is feature freeze, until the next stable release. Only bug fixes go in.
I don't want to introduce last minute command-line arguments that have to be propagated everywhere, and last-minute regressions in loading existing trained models.

**Comment by Mehrad0711** [comment]
Okay then, this PR can wait...
There was a bug in predict.py though which I will send a separate PR for.

File: genienlp/predict.py [hunk 1]
```diff
@@ -55,11 +55,11 @@ def get_all_splits(args):
...
         kwargs = {'train': None}
         if args.evaluate == 'valid':
-            kwargs['validation'] = None
-        elif args.evaluate == 'test':
             kwargs['test'] = None
+        elif args.evaluate == 'test':
+            kwargs['validation'] = None
         else:
-            raise ValueError('Validation split should be either valid or test')
+            raise ValueError('Split used for prediction should be either valid or test')
```
Comment (Mehrad0711) [inline genienlp/predict.py]: @gcampax

**Comment by gcampax** [comment]
Hey @Mehrad0711 can you make a clean PR with just the fix?

**Comment by Mehrad0711** [comment]
Here: #13

**Comment by gcampax** [comment]
Release went out, are you going to merge this?

**Comment by Mehrad0711** [comment]
ah just saw your comment here. I want to make sure it works completely before merging. Maybe another day or two...

**Pull request merged**